### ISSUE TITLE
consider aliasing className

### ISSUE THREAD

**Issue by substack** [dekujs/deku#411]
Upstream in hyperx land, I merged a patch so that people won't have to use `className` in their template strings, but this broke the deku example for using `class=` in the code which is now aliased to className, which all the other hyperscript-style APIs appear to support. Several of the other APIs only support html element attributes like `className`, so I really can't win here without bothering somebody.

Would deku folk be open to aliasing `className` to `class`? Not because people should be writing `className` themselves, but because it helps with interoperability with the rest of the ecosystem.

Related: https://github.com/substack/hyperx/pull/22

**Comment by rstacruz** [comment]
I have no opinion on the matter, but curious: have you tried asking React if they'd be open to aliasing `class` as ` className`?

**Comment by anthonyshort** [comment]
Yeah this is fine by me. It's a really small change for us. The React people would probably have a ton of stuff to update.
